core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -364,14 +359,8 @@ var TopicsAndSkillsDashboardPage = function() {
       '.protractor-test-unassign-topic-label', topicName));
     var assignedTopicInput = topicListItem.element(
       assignedTopicNameInputClass);
-    await waitFor.elementToBeClickable(
-      assignedTopicInput,
-      'Assigned topic checkbox takes too long to be clickable');
-    await assignedTopicInput.click();
+    await action.click('Assigned Topic Checkbox', assignedTopicInput);
 
-    await waitFor.elementToBeClickable(
-      confirmUnassignSkillButton,
-      'Confirm Unassign skill button takes too long to be clickable');
     await action.click('Confirm Unassign Skill', confirmUnassignSkillButton);
     await waitFor.invisibilityOf(
       confirmUnassignSkillButton,
```

Comment:
Below in `this.expectNumberOfTopicsToBe`, please add a waitFor before the `.count()`. You'll need to wrap your waitFOr in an `if` block so it only runs when we expect topics to be present. Please also add waitFor statements before `.getText()` calls and before `expect(await topicNames.count()).toEqual(1);`.